README.md

```diff
@@ -75,7 +75,95 @@ More examples, as well as an explanation of input/output are available in [offic
 
 ### Using command line
 
-TODO: add this section.
+#### Downloading Embeddings
```

Comment:
makes sense to include the step with installing `extra-model` first?

Reply:
good point, added to CLI & package sections